spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'GET/show' do
+  context 'with success response' do
+    it 'should return success' do
+      get '/users/:user_id/posts'
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('Here is a list of posts with comments for a given user')
+    end
+
+    it 'should return success' do
+      get '/users/:user_id/posts/:id'
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('Post details for a given user')
```

Comment:
- According to the project requirements, for each action, you should check if the correct template is rendered, but this is missing in your test. Please correct this for **all** action tests 🙏